internal/cli/opsmanager/clusters/list.go

```diff
@@ -37,21 +37,46 @@ func (opts *ListOpts) init() error {
 	return err
 }
 
+// listTemplate used when project ID is given
+var listTemplate = `ID	NAME	TYPE{{range .Results}}
+{{.ID}}	{{.ClusterName}}	{{.TypeName}}{{end}}
+`
+
+// listAllTemplate used fetching all clusters for all projects
+var listAllTemplate = `ID	NAME	TYPE{{range .Results}}{{range .Clusters}}
+{{.ClusterID}}	{{.Name}}	{{.Type}}{{end}}{{end}}
+`
+
 func (opts *ListOpts) Run() error {
+	r, err := opts.clusters()
+	if err != nil {
+		return err
+	}
+
+	return output.Print(config.Default(), opts.template(), r)
```

Comment:
Could also potentially choose the template based on a type switch from the interface, but I don't think it's ideal

Reply:
I just wasn't sure on returning three values from that other function